misc/do_nightly_build

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
-#we set the compiledir to the /Tmp dir to make the test faster by bypassing the nfs network.
+
+# If not jenkins, set workspace to local Tmp
+if [ -v $WORKSPACE ]; then
+   WORKSPACE=/Tmp
```

Comment:
This isn't good. Most computer don't have /Tmp. if WORKSPACE isn't defined, and TMPDIR is defined, use TMPDIR else use /tmp.